src/email/templates/Verify/Verify.tsx

```diff
@@ -1,31 +1,22 @@
 import React from 'react';
 
-import { getProfileUrl } from '@/server/lib/getProfileUrl';
-import { Routes } from '@/shared/model/Routes';
 import { Page } from '@/email/components/Page';
 import { Button } from '@/email/components/Button';
 import { Header } from '@/email/components/Header';
 import { SubHeader } from '@/email/components/SubHeader';
 import { Text } from '@/email/components/Text';
 import { Footer } from '@/email/components/Footer';
 
-const profileUrl = getProfileUrl();
-
 export const Verify = () => {
   return (
     <Page>
       <Header />
-      <SubHeader>Please verify your email</SubHeader>
+      <SubHeader>Thank you for $registerAction</SubHeader>
```

Comment:
This seems to be introducing a new convention for how text is replaced. Other areas of the code use upper case text like `TOKEN_PLACEHOLDER`. 

I'm happy to go either way but I think we should be consistent.

Reply:
As mentioned on chat. This is only temporary while we still use IDAPI to send the emails. When we move over to using gateway (or another email service) we can use one way or another on how text is replaced.